todoapp/users/models.py

```
@@ -1,30 +1,50 @@
 
+    first_name = models.CharField("first_name", max_length=30)
+    last_name = models.CharField("last_name", max_length=150, blank=True)
+    email = models.EmailField("email address", unique=True)
+    is_staff = models.BooleanField(
+        "staff status",
+        default = False,
+        help_text="user can login to admin site or not."
+    )
+    date_joined = models.DateTimeField(
+        "date joined", 
+        auto_now_add=True
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)